src/pathfinding_service/database.py

```diff
@@ -203,3 +204,27 @@ def delete_channel_views(self, channel_id: ChannelID) -> None:
     def get_token_networks(self) -> Iterator[TokenNetwork]:
         for row in self.conn.execute("SELECT address FROM token_network"):
             yield TokenNetwork(token_network_address=decode_hex(row[0]))
+
+    def insert_feedback_token(self, token: FeedbackToken) -> None:
+        token_dict = dict(token_id=token.id.hex, expiry=token.expiry)
+        self.conn.execute(
+            """
+            INSERT INTO feedback_token (
+                token_id, expiry
```

Comment:
I would rather store the time at which the token is generated. This seems like the more meaningful time to me. Should work either way, though.